src/styles/StyleUtils.ts

```diff
@@ -462,7 +462,7 @@ function getBorderColorStyle(borderColor: string): ViewStyle {
 /**
  * Returns the width style for the wordmark logo on the sign in page
  */
-function getSignInWordmarkWidthStyle(environment: string, isSmallScreenWidth: boolean): ViewStyle {
+function getSignInWordmarkWidthStyle(environment: ValueOf<typeof CONST.ENVIRONMENT> | undefined, isSmallScreenWidth: boolean): ViewStyle {
```

Comment:
Is there a shorthand way to set the param type here? Asking as I'm new to Typescript but familiar with Kotlin/Swift.

Also, is making this optional really necessary? It looks like the only usage of this function will reliably return an Environment... or is this due to a potential issue with `EnvironmentContext`? Thanks!

Reply:
@Julesssss We can switch `environment` and `isSmallScreenWidth` params  places and then mark `environment` param as optional this way:
```
function getSignInWordmarkWidthStyle(isSmallScreenWidth: boolean, environment?: ValueOf<typeof CONST.ENVIRONMENT>): ViewStyle {
```
Yeah, since [`EnvironmentContext` can be `null`](https://github.com/Expensify/App/blob/main/src/components/withEnvironment.tsx#L22), it's better to keep it optional.

Reply:
Okay thanks. 

I'm not too bothered about the param ordering, but `environment?:` seems a bit cleaner to me. But if this goes against typical formatting rules please ignore me

Reply:
@Julesssss Updated 🙂

Reply:
@fabioh8010 @Julesssss @VickyStash Shouldn't environment be required? I think we should throw an exception instead of returning an empty object in [useEnvironment](https://github.com/Expensify/App/blob/db37e46f58aceea23c0136e1bdce6e3ed9d2d90f/src/hooks/useEnvironment.ts#L12) because EnrivonmentProvider is passed in [root app](https://github.com/Expensify/App/blob/db37e46f58aceea23c0136e1bdce6e3ed9d2d90f/src/App.js#L64C21-L64C40). So that will allow us to remove possible null from getSignInWordmarkWidthStyle. I think there is no case when the environment is null.
What do you think about this approach?

Reply:
@ArekChr yeah if it is possible to enforce a non-optional environment, that sounds good to me. Happy to review a PR if you could raise one